web/style.css

```diff
@@ -120,12 +698,55 @@ h1, #msg-inner {
     background: white;
 }
 
+#load {
+    display: grid;
+    
+    width: 100%;
+    background-color: #ddd;
+
+    border-radius: 4px;
+    border: solid 1px black;
+}
+
+#load-bar {
+    grid-column: 1;
+    grid-row: 1;
+
+    width: 0%;
+    height: 30px;
+    background-color: #04AA6D;
+}
```

Comment:
We experimented with adding a `transition: width 500ms` here, this gives a nice effect.